string folder
string model_filename
---

